Installing Bolt 6.0.0-beta.1 with SQLite fails

FYI. for @bobvandevijver and @macintoshplus 

Normally when running the composer create project command and choosing SQLite for the database, the installation concludes with a filled (this is optional) SQLite database.

However when attempting to install the beta like so (installing in a premade directory):

`composer create-project bolt/project . --stability=beta`

You end up with these error messages

<img width="2542" height="986" alt="Image" src="https://github.com/user-attachments/assets/5bc4c53a-f00f-4de2-aaf6-b655c8eb6c72" />

and fails.

Note this is just the SQLite database. All PHP packages installed withouth any error messages. Used PHP 8.2.x

If I then manually force a migration. And re-run `bin/console bolt:setup` to install fixtures. Installation finishes.

BUT. Cannot login in to the backend dashboard whatsoever.

I checked when listing Bolt users. Login will simply not work.

So, this is doctrine being doctrine I guess. The migration is generated, but the subsequent command to execute fails silently (due to a null output being passed).

```
bobv@XPS15-9520-Bob:/tmp/bolt-project$ bin/console bolt:setup
 Generated new migration class to "/tmp/bolt-project/migrations/Version20251206092350.php"

 To run just this migration for testing purposes, you can use migrations:execute --up "DoctrineMigrations\\Version20251206092350"

 To revert the migration you can use migrations:execute --down "DoctrineMigrations\\Version20251206092350"

 [ERROR] The version "latest" couldn't be reached, there are no registered migrations.
```

--- a/phpstan-baseline.php
+++ b/phpstan-baseline.php
@@ -634,7 +634,7 @@ $ignoreErrors[] = [
 $ignoreErrors[] = [
 	'message' => '#^Cannot call method find\\(\\) on Symfony\\\\Component\\\\Console\\\\Application\\|null\\.$#',
 	'identifier' => 'method.nonObject',
-	'count' => 7,
+	'count' => 3,
 	'path' => __DIR__ . '/src/Command/SetupCommand.php',
 ];
 $ignoreErrors[] = [

--- a/src/Command/SetupCommand.php
+++ b/src/Command/SetupCommand.php
@@ -4,12 +4,12 @@ declare(strict_types=1);
 
 namespace Bolt\Command;
 
+use Bolt\ComposerScripts\Script;
 use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\NullOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
@@ -29,27 +29,23 @@ class SetupCommand extends Command
     {
         $io = new SymfonyStyle($input, $output);
 
-        $command = $this->getApplication()->find('doctrine:migrations:sync-metadata-storage');
-        $exitCode = $command->run(new ArrayInput([]), new NullOutput());
+        $exitCode = Script::runConsole(['doctrine:migrations:sync-metadata-storage']);
         $this->processExitCode($exitCode, 'An error occurred when initialising the Doctrine Migrations metatada storage.');
 
-        $command = $this->getApplication()->find('doctrine:migrations:diff');
-        $exitCode = $command->run(new ArrayInput([
-            '--from-empty-schema' => true,
-            '--no-interaction' => true,
-        ]), $output);
+        $exitCode = Script::runConsole([
+            'doctrine:migrations:diff',
+            '--from-empty-schema',
+            '--no-interaction',
+        ]);
         $this->processExitCode($exitCode, 'An error occurred when creating the initial migration.');
 
-        $command = $this->getApplication()->find('doctrine:migrations:migrate');
-        $commandInput = new ArrayInput([
-            '--no-interaction' => true,
+        $exitCode = Script::runConsole([
+            'doctrine:migrations:migrate',
+            '--no-interaction',
         ]);
-        $commandInput->setInteractive(false);
-        $exitCode = $command->run($commandInput, new NullOutput());
         $this->processExitCode($exitCode, 'An error occurred while executing the initial migration.');
 
-        $command = $this->getApplication()->find('bolt:reset-secret');
-        $exitCode = $command->run(new ArrayInput([]), new NullOutput());
+        $exitCode = Script::runConsole(['bolt:reset-secret']);
         $this->processExitCode($exitCode, 'An error occurred while resetting APP_SECRET in the .env file.');
 
         $command = $this->getApplication()->find('bolt:add-user');

